Farm&Gun/Assets/Scripts/Entities/Opponents/GenerateOpponent.cs

```diff
@@ -6,22 +6,58 @@ public class GenerateOpponent : MonoBehaviour
     [SerializeField]
     private GameObject wolf, fox;
 
-    [SerializeField]
     private List<GameObject> wolfs, foxes;
 
+    [SerializeField]
+    private int amountWolf, amountFox;
+    
+    private void Awake()
+    {
+        wolfs = new ();
+        foxes = new ();
```

Comment:
Won't this create errors?

Reply:
above is type definition, this is just initialization, type will be automaticly resolved